demo/owner/README.md

```diff
@@ -180,4 +180,32 @@ The PKCS12 keystore file \<fido-iot-src\>/demo/owner/owner_keystore.p12 contains
 
 # Troubleshooting
 
-As the H2 DB grows, larger heap space will be required by the application to run the service. Default configured heap size is `256 MB`. Increase the heap size appropriately in `demo/owner/owner-entrypoint.sh` to avoid heap size issues.
\ No newline at end of file
+As the H2 DB grows, larger heap space will be required by the application to run the service. Default configured heap size is `256 MB`. Increase the heap size appropriately in `demo/owner/owner-entrypoint.sh` to avoid heap size issues.
+
+# Configuring Owner for HTTPS/TLS Communication
+
+By default, the Owner uses HTTP for all communications on port 8042. In addition to that, the Owner can be configured to handle HTTPS request from the device.
+
+- Generate the Keystore/Certificate for the Owner
+
+  * Ensure that the web certificate is issued to the resolvable domain of the Owner server.
+
+- Copy the generated Keystore/Certificate to `demo/owner/certs` folder.
+
+- Copy the truststore containing all the required certificates to `demo/owner/certs` folder.
+
```

Comment:
Add instructions to update RV blob for HTTPS

Reply:
Done.